### PR TITLE
Scaling Optimization for Dashboard Page

### LINKED ISSUE #3097: Dashboard mobile scaling issue
**Actual Behaviour**

There is a weird scroll area inside the dashboard where bots are displayed and the scalling of these sections's content in mobile isn't that good. 
There is a lot area being wasted in padding which can be used to display more content efficiently
susi ai_dashboard(iPhone 6_7_8)

**Expected Behavior**

The user shouldn't be frustrated scrolling left and right most of the times and the page should scale relative to mobile screen size.

**Steps to reproduce it**

- open https://susi.ai/dashboard
- go to dev tools and toggle device toolbar
Alternatively 
-open the https://susi.ai/dashboard in mobile

**Would you like to work on the issue?**

Yes I am working on this issue! 😇

### PR DESCRIPTION
**Pull request by Sid911**
Fixes #3097 

Changes: Changed padding and added dynamic removal of first and last page button from pagination depending on the screen size

Demo Link: https://pr-3101-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 
- When screen width is above 500px
image
- When screen width is below 500px
image

### CODE REVIEW

File: src/components/cms/Dashboard/MySkills.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ const Container = styled.div`
     margin-right: 0;
   }
 `;
-
+const issmallscreen = window.screen.availWidth < 500;
```
Comment (akshatnitd) [inline src/components/cms/Dashboard/MySkills.js]: Use camel case for variable naming. `isSmallScreen`

Comment (Sid911) [reply src/components/cms/Dashboard/MySkills.js]: I used _(underscores) before linters give errors on that. I should have used CamelCase.. Thanks for pointing that out. Will Do that change

**Comment by Sid911** [comment]
@akshatnitd Done 😇

File: src/components/cms/Dashboard/MySkills.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ const Container = styled.div`
     margin-right: 0;
   }
 `;
-
+const IsSmallScreen = window.screen.availWidth < 500;
```
Comment (akshatnitd) [inline src/components/cms/Dashboard/MySkills.js]: According to camelCase, it should be `isSmallScreen`.

**Comment by Sid911** [comment]
@akshatnid Done ✌️... I was just wondering why can't we use underscores, because in c++projects like somewhat of direct X and winAPI based namings _ is very common to see... Plus Microsoft has m_ naming conventions etc...

Is it like "project code consistency" or is there some technical reason?

**Comment by Sid911** [comment]
@akshatnitd  Done ✌️... I was just wondering why can't we use underscores, because in c++projects like somewhat of direct X and winAPI based namings _ is very common to see... Plus Microsoft has m_ naming conventions etc...

Is it like "project code consistency" or is there some technical reason in web?

**Comment by akshatnitd** [comment]
@Sid911 The main reason is following conventions and consistency across  language/libraries.

**Comment by Sid911** [comment]
Done @akshatnitd @plxity

**Comment by Sid911** [comment]
@akshatnitd @Orbiter can this be merged please .. I am running out of time in terms of completeting the task

**Pull request merged**